amitools/vamos/lib/ExecLibrary.py

```diff
@@ -370,9 +392,10 @@ def FreeMem(self, ctx):
             log_exec.info("FreeMem: 0x%06x %d bytes -> %s" % (addr, size, mb))
             self.alloc.free_memory(mb)
         else:
-            raise VamosInternalError(
-                "FreeMem: Unknown memory to free: ptr=%06x size=%06x" % (addr, size)
-            )
+            pass
+             #raise VamosInternalError(
```

Comment:
Maybe replace with log_exec.info() instead of commenting out?

Reply:
Arg sorry. It should just be uncommented and left as it was! It was early tests with a AllocAbs I forgot to restore the old code.

Reply:
you can update this PR by committing to your branch @jotd666, no need to create a new PR.